src/vlt/src/commands/list.ts

```diff
@@ -55,17 +52,20 @@ export const usage: CommandUsage = () =>
 
 export type ListResult = JSONOutputGraph &
   MermaidOutputGraph &
-  HumanReadableOutputGraph
+  HumanReadableOutputGraph & { queryString: string }
 
-export const views: Views<ListResult> = {
-  /* c8 ignore next */
-  defaultView: process.stdout.isTTY ? 'human' : 'json',
-  views: {
-    json: jsonOutput,
-    mermaid: mermaidOutput,
-    human: humanReadableOutput,
+export const views = {
+  json: jsonOutput,
+  mermaid: mermaidOutput,
+  human: humanReadableOutput,
+  gui: async ({ queryString }, _, conf) => {
+    await startGUI({
+      conf,
+      startingRoute:
+        '/explore?query=' + encodeURIComponent(queryString),
+    })
   },
```

Comment:
Yes! This is what I had envisioned originally but couldn't get it play nicely at the time with output. But I think this is one example that shows this whole PR is the refactor in the right direction.